include/tvm/auto_scheduler/search_policy.h

```diff
@@ -89,46 +100,54 @@ class SearchCallback : public ObjectRef {
   TVM_DEFINE_MUTABLE_OBJECT_REF_METHODS(SearchCallback, ObjectRef, SearchCallbackNode);
 };
 
+/*! \brief Attribute keys of ops used for SearchPolicy. */
+struct SearchPolicyKey {
+  /*! \brief Always apply unroll to the inner most iterator of the specificed iterators. */
+  static constexpr const char* always_unroll_inner = "auto_scheduler_always_unroll_inner";
+  /*! \brief The specified iterators will not be placed as the inner most iterator. */
+  static constexpr const char* no_split_at_inner = "auto_scheduler_no_split_at_inner";
+  /*! \brief The specified iterators will not be placed as the outter most iterator. */
+  static constexpr const char* no_split_at_outer = "auto_scheduler_no_split_at_outer";
```

Comment:
I think we will no longer need these two attributes now. These two attributes only needed in the original winograd implementation.

Reply:
Ok, then I'll remove them.

Reply:
"auto_scheduler_no_split_at_inner" is useful for some sparse operators. We can keep it.